# Ardrone3PilotingStateAttitudeChanged
# auto-generated from https://raw.githubusercontent.com/Parrot-Developers/libARCommands/7e2f55fafcd45ba2380ca2574a08b7359c005f47/Xml/ARDrone3_commands.xml
# Do not modify this file by hand. Check scripts/meta folder for generator files.
#
# SDK Comment: Drone attitude changed

Header header

# roll value (in radian)
float32 roll
# Pitch value (in radian)
float32 pitch
# Yaw value (in radian)
float32 yaw
